internal/app/machined/pkg/runtime/v1alpha1/bootloader/grub/install.go

```diff
@@ -39,11 +39,6 @@ func (c *Config) Install(opts options.InstallOptions) (*options.InstallResult, e
 				FilesystemType: partition.FilesystemTypeXFS,
 				MountTarget:    filepath.Join(opts.MountPrefix, constants.BootMountPoint),
 			},
-			{
-				PartitionLabel: constants.EFIPartitionLabel,
```

Comment:
what if we are upgrading 1.9->1.10, and GRUB is used for EFI?

Reply:
hmmm, I see, so this needs to be based on host talos version

Reply:
yeh, did that only, and verified the upgrade path